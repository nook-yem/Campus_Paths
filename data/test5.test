r
East Campus Athletic Village Arena
89
r
Blitman Residence Commons
Blitman Residence Commons
r
75
75
r
74
CBIS
q
