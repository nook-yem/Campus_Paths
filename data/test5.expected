First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from East Campus Athletic Village Arena to East Campus Athletic Village Arena:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Blitman Residence Commons to Blitman Residence Commons:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Parking Garage to Parking Garage:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from CBIS to CBIS:
Total distance: 0.000 pixel units.
